src/values/FunctionValue.js

```diff
@@ -13,25 +13,17 @@ import type { ObjectKind } from "../types.js";
 import type { LexicalEnvironment } from "../environment.js";
 import type { Realm } from "../realm.js";
 import { ObjectValue, NumberValue } from "./index.js";
-import * as t from "babel-types";
-import type { BabelNodeLVal, BabelNodeBlockStatement, BabelNodeSourceLocation } from "babel-types";
 import invariant from "../invariant.js";
 
+/* Abstract base class for all function objects */
 export default class FunctionValue extends ObjectValue {
   constructor(realm: Realm, intrinsicName?: string) {
     super(realm, realm.intrinsics.FunctionPrototype, intrinsicName);
   }
 
   $ConstructorKind: void | string;
```

Comment:
That is my initial thought too. Unfortunately, I found NativeFunctionValue actually use it here:
https://github.com/facebook/prepack/blob/4824b47038131efcfd3e5f3928e2d2833a2aac23/src/values/NativeFunctionValue.js#L51
And this NativeFunctionValue.$ConstructorKind is later used in InternalConstruct() in function.js.
So we can't move it down to ECMAScriptFunctionValue.